spring-ai-docs/src/main/antora/modules/ROOT/pages/api/index.adoc

```diff
@@ -6,9 +6,9 @@ The Spring AI API covers a wide range of functionalities.
 Each major feature is detailed in its own dedicated section.
 To provide an overview, the following key functionalities are available:
 
-* Portable API across AI providers for Chat, Text to Image, and Embedding models. Both synchronous and stream API options are supported. Dropping down to access model specific features is also supported.  We support AI Models from OpenAI, Microsoft, Amazon, Google, Huggingface and more.
-* Portable API across Vector Store providers, including a novel SQL-like metadata filter API that is also portable.  Support for 8 vector databases are available.
-* Function calling.  Spring AI makes it easy to have the AI model invoke your POJO java.util.Function object.
+* Portable API across AI providers for Chat, Text to Image, Audio Transcription, Text to Speech, and Embedding models. Both synchronous and stream API options are supported. Dropping down to access model specific features is also supported.  We support AI Models from OpenAI, Microsoft, Amazon, Google, Huggingface and more.
+* Portable API across Vector Store providers, including a novel SQL-like metadata filter API that is also portable. Support for 14 vector databases are available.
```

Comment:
I'm not sure if we want to continue having the total number of supported vector databases here. In any case, it's now reflecting the number we have today.